test/integration/cli/reporter-csv.test.js

```diff
@@ -20,10 +20,10 @@ describe('CLI reporter CSV', () => {
 
 		it('outputs issues in CSV format', () => {
 			const lines = pa11yResponse.output.trim().split('\n');
-			assert.lengthEquals(lines, 5);
+			assert.lengthEquals(lines, 18);
 			assert.strictEqual(lines[0], '"type","code","message","context","selector"');
 			lines.slice(1).forEach(line => {
-				assert.match(line, /^"(error|warning|notice)","[^"]+","[^"]+",".*","[^"]+"$/i);
+				assert.match(line, /^"(error|warning|notice)","[^"]+","[^"]+",(".*"|null),"[^"]*"$/i);
```

Comment:
The CSV output shouldn't contain `null` values. We might need to add some pre-formatting to convert null to empty strings or `\n` etc

Reply:
Should that be done in a different PR, but before next release?

Reply:
My worry that is that this will introduce a breaking change for those who rely on the CSV output.

Reply:
Looks like csv reporting is generated in another project, namely `pa11y-reporter-csv`. How to solve?

Reply:
Without delving into `pa11y-reporter-csv`, I would hazard a guess that for notices the `context` is now `null` and it doesn't print that out nicely.
We can either `map` our results to use an empty string rather than `null`, or update the reporter to handle `null` values instead.

Reply:
I've looked at this and it just seems that previously there was nothing that would have a `null` context. Theres a test case for `context` being null as well. So this is desired